draft-ietf-cdni-ci-triggers-rfc8007bis.xml

```diff
@@ -3084,49 +3090,86 @@ dCDN MAY reject such requests for urgent processing using “ereject” error co
       </t>
 
 
-     <section title="CI/T Objects Versions Capability Object">
+     <section title="CI/T Trigger Endpoint Capability Object" anchor="cit-trigger-endpoints-capability-object">
         <t>
-           The CI/T objects versions capability object is used to indicate support for one or more CI/T objects versions.
-		   Capability type is "FCI.CITObjectsVersion".
-		   For simplicity, it is recommended to minimize the set of supported versions declared by the dCDN. For example, one way to minimize supported versions is to support only objects of version 2.
-           Version 1, as originally defined in <xref target="RFC8007"/>, is the default if this capability is not explicitly declared.
+		The CI/T triger endpoint capability object is used to advertise one or more CI/T interface endpoints 
```

Comment:
triger -> trigger

Reply:
For consistency and to reduce redundancy, should we call this the CI/T Endpoints capability?